javascript/javascript3/week2/Task4_Fetching_Waiting/fetching_waiting.js

```diff
@@ -0,0 +1,38 @@
+// TASK 4 Fetching and Waiting
+/*  The 3 steps:
+    Wait 3 seconds
+    After 3 seconds fetch some data from any api you like
+    Log out the data from the api */
+
+// USING PROMISES AND .then
+
+url = "https://api.chucknorris.io/jokes/random";
```

Comment:
`const url = "https://api.chucknorris.io/jokes/random";`

Reply:
Yes, you are damn right! I had this code together with the other folders so const url was already defined somewhere else. 
I have changed it to const url now.